monome/teletype
# jf4.0 ops, maths docs fix

#### What does this PR do?
Adds JF4.0 OPs and expands existing OPs where they might address channels on two concurrent JF.
Fixes a docs error for maths section from earlier PR (missing quote encapsulation).

#### Provide links to any related discussion on [lines](https://llllllll.co/).
https://llllllll.co/t/teletype-3-2-feature-requests-and-discussions/32052/297

#### How should this be manually tested?
Attach a single JF and run `JF.ADDR 2` to move it to the 2nd JF address. Attach another JF which will remain on primary address.

`JF.SEL 1` or `2` should cause subsequent JF commands to affect the specified JF unit, as should the MODs `JF1:...` and `JF2:...`. Note that the selected address is treated as "primary" by subsequent OPs.
`JF0:...` MOD should cause the following command to execute on both JF units.

Knob and switch values getters (`JF.RAMP`, `JF.CURVE`, `JF.FM`, `JF.TSC`, `JF.SPEED`, `JF.INTONE`, and `JF.TIME`) should return values from the previously selected JF unit. `JF.TIME` and `JF.INTONE` will add their CV offsets as well.

Expanded channel addressing is: 0 = all channels primary JF, -1 = all channels both JF, 1..6 = primary channels, 7..12 = secondary channels. So `JF.TR -1 1` should fire all triggers on both units and `DEL.R 12 100: JF.TR + I 1 1` should sweep triggers across all 12 channels.

With `O.MIN 0` and `O.MAX 11`, calling `JF.POLY N O V 10` successively should make a chromatic walk-up. Notes will automatically switch between primary and secondary JF every 6 notes, regardless of how many voices are in use.

#### Any background context you want to provide?
I used [jf.lua](https://github.com/monome/crow/blob/main/lua/ii/jf.lua) as a guide for the I2C commands.
Unable to run "make format" in the Docker image (I hope to learn soon how to modify the image to include CLANG-FORMAT).

#### If the related Github issues aren't referenced in your commits, please link to them here.
N/A

#### I have,
